pkg/utils/kubelessutil.go

```diff
@@ -406,11 +407,14 @@ func populatePodSpec(funcObj *kubelessApi.Function, lr *langruntime.Langruntimes
 	}
 	if funcObj.Spec.Deps != "" && err != nil {
 		return fmt.Errorf("Unable to install dependencies for the runtime %s", funcObj.Spec.Runtime)
-	} else if funcObj.Spec.Deps != "" {
+	} else if funcObj.Spec.Deps != "" || funcObj.Spec.UseBundledDeps != "" {
 		depsChecksum, err := getChecksum(funcObj.Spec.Deps)
 		if err != nil {
 			return fmt.Errorf("Unable to obtain dependencies checksum: %v", err)
 		}
+		if funcObj.Spec.UseBundledDeps != "" {
+			depsChecksum = ""
+		}
```

Comment:
nitpick: I would do the opposite, define `depsChecksum` to "" and get the checksum only if `funcObj.Spec.Deps` is not empty.